### DISCUSSION TITLE
global Skript

### DISCUSSION THREAD

**Discussion by GothaBauer** [Mogli12/KeyboardSteer#741] [Ideas]
mal eine Frage .
Kannst Du ein Skript entwickeln das global jeden Traktor ein Lastschaltgetriebe hinzufügt ?
Im Moment füge ich den aus den Editor erstellten Mods ein E_23 getiebe über die xml hinzu .
Das ist leider sehr aufwendig .
Ich weiß das ist nicht immer Real aber bei einigen Arbeiten zB. grubber mulchen usw . Soundtechnik besser wenn man den Gang und Geschwindigkeit festlegen kann .
Danke für deine bisherige Arbeit mach weiter so

**Comment by Mogli12** [comment]
Du kannst mit VCA die Drehzahl vorgeben (Handgas).
